### ISSUE TITLE
Portal loop from cli: Signature verification failed; verify correct account, sequence, and chain-id

### ISSUE THREAD

**Issue by grepsuzette** [gnolang/gno#2109]
**tl;dr** Regardless of which way I use, I get this error: `signature verification failed; verify correct account, sequence, and chain-id`.

I'm trying to interact with the portal loop, calling `Register` from `/r/demo/microblog`. 
Following the instructions from https://gno.land/r/demo/microblog?help:

### INSECURE BUT QUICK

 `gnokey maketx call -pkgpath "gno.land/r/demo/microblog" -func "Register" -gas-fee 1000000ugnot -gas-wanted 2000000 -send "" -broadcast -chainid "portal-loop" -args "grepsuzette" -args "once in a previous life he lived in a hut * once in present times he also ate in a pizza hut" -remote "https://rpc.gno.land:443" g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d`

```
--= Error =--
Data: unauthorized error
Msg Traces:
    0  /home/bob/GNO/gno/tm2/pkg/crypto/keys/client/maketx.go:174 - check transaction failed: log:--= Error =--
Data: std.UnauthorizedError{abciError:std.abciError{}}
Msg Traces:
    0  /opt/build/tm2/pkg/std/errors.go:70 - signature verification failed; verify correct account, sequence, and chain-id
```

### FULL SECURITY WITH AIRGAP

`gnokey query -remote "https://rpc.gno.land:443" auth/accounts/g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d
height: 0`

```json
data: {
  "BaseAccount": {
    "address": "g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d",
    "coins": "30000000ugnot",
    "public_key": null,
    "account_number": "173",
    "sequence": "0"
  }
}
```

* `gnokey maketx call -pkgpath "gno.land/r/demo/microblog" -func "Register" -gas-fee 1000000ugnot -gas-wanted 2000000 -send "" -args "grepsuzette" -args "once in a previous life he has lived in a hut * once in the present times he also ate in a pizza hut" g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d > unsigned.tx`
* `gnokey sign -txpath unsigned.tx -chainid "portal-loop" -number 173 -sequence 0 g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d > signed.tx`
* `gnokey broadcast -remote "https://rpc.gno.land:443" signed.tx`

`--= Error =--`
`Data: transaction failed &core_types.ResultBroadcastTxCommit{CheckTx:abci.ResponseCheckTx{ResponseBase:abci.ResponseBase{Error:std.UnauthorizedError{abciError:std.abciError{}}, Data:[]uint8(nil), Events:[]abci.Event(nil), Log:"--= Error =--\nData: std.UnauthorizedError{abciError:std.abciError{}}\nMsg Traces:\n    0  /opt/build/tm2/pkg/std/errors.go:70 - signature verification failed; verify correct account, sequence, and chain-id\nStack Trace:`

Is there anything special making it not work? I haven't used the portal loop before.

**Comment by thehowl** [comment]
Have you updated gnokey?

(#1939 made a backwards incompat change which leads to this error)

**Comment by grepsuzette** [comment]
Yep that was exactly it! Thanks
